nuxeo-jsf/nuxeo-platform-webapp-base/src/main/java/org/nuxeo/ecm/webapp/tree/BigFolderLeafFilter.java

```diff
@@ -31,13 +32,11 @@
 
     private static final long serialVersionUID = 1L;
 
-    public static final String BIG_FOLDER_FACET = "BigFolder";
-
```

Comment:
a detail: since this is a public String, a good practice is to keep it, deprecate it stating the deprecation version and referencing the new variable (and make it point to it for compat)